modules/ROOT/pages/ftp-server-cookbook.adoc

```diff
@@ -92,7 +82,7 @@ After configuring the FTP server you can run the test:
 </munit:test>
 ----
 
-This FTP accepts any user, so there is no need to set up a user database or list.
+This FTP accepts any user, so you must not set up a user database or list.
```

Comment:
I like the previous version, because "you must" sounds like there's some condition for running this.

Reply:
This modification was made to get the content aligned with our style guide.